clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -7976,6 +7976,8 @@ void CodeGenModule::requireVectorDestructorDefinition(const CXXRecordDecl *RD) {
     auto *NewFn = llvm::Function::Create(
         cast<llvm::FunctionType>(VDEntry->getValueType()),
         llvm::Function::ExternalLinkage, VDName, &getModule());
+    SetFunctionAttributes(VectorDtorGD, NewFn, /*IsIncompleteFunction*/ false,
+                          /*IsThunk*/ false);
```

Comment:
Nice find!

Can you add a test that covers this?

Reply:
Oh, I actually added a test for that, just forgot to hit git add. Here it is https://github.com/llvm/llvm-project/pull/133451/commits/dd4f396c36804c328996ed69ea0388956857b4f1